src/Sidekick.Apis.Poe/ITradeFilterService.cs

```diff
@@ -25,5 +25,5 @@ public interface ITradeFilterService
     /// </summary>
     /// <param name="item">The item for which to get property filters.</param>
     /// <returns>The property filters.</returns>
-    Task<PropertyFilters> GetPropertyFilters(Item item);
+    PropertyFilters GetPropertyFilters(Item item);
```

Comment:
Leave this as a task. I have some changes coming that require reading from the settings to set the default filters.